main.py

```diff
@@ -2,6 +2,42 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
+
+GENRES = [
+    "Western",
+    "Action",
+    "Dramma",
+]
+ACTORS = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson"),
+]
+
 
 def main() -> QuerySet:
-    pass
+    for genre in GENRES:
+        Genre.objects.create(name=genre)
+
+    for first_name, last_name in ACTORS:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
```

Comment:
You can break down main function in smaller specialized functions to structure your code better. E.g separate function for creating, updating, deleting..